config.go

```diff
@@ -147,6 +147,8 @@ var (
 // all config items of its enveloping subservers, each prefixed with their
 // daemon's short name.
 type Config struct {
+	ShowVersion bool `long:"version" description:"Display version information and exit"`
```

Comment:
ultra-nit: looks like the other entries in this section have a full stop at the end of their description...